src/thor/triplegbuilder.cc

```diff
@@ -747,6 +747,37 @@ void SetElevation(TripLeg_Edge* trip_edge,
   }
 }
 
+void ProcessNonTaggedValue(valhalla::StreetName* trip_edge_name,
```

Comment:
Just moved this function further to the top since it's now needed in `AddTripIntersectingEdge´